src/server/game/Entities/Object/Object.cpp

```diff
@@ -997,6 +997,80 @@ void MovementInfo::OutDebug()
         TC_LOG_DEBUG("misc", "splineElevation: %f", splineElevation);
 }
 
+void MovementInfo::WriteContentIntoPacket(ByteBuffer * data, bool includeGuid /* = false*/) const
```

Comment:
I won't make this refactoring a part of this PR. This has nothing to do with my PR and it's way too much work. 'WriteContentIntoPacket' can be renamed to 'operator<<' once the refactoring is done. This is just cosmetic stuff.